test/node/parseIDLFileDAO-helper.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+global.parseIDLFileDAOTest = function(data) {
+  describe(data.description, function() {
+    var execSync = require('child_process').execSync;
+    var originalTimeout;
+
+    afterAll(function() {
+      execSync(`/bin/rm -rf "${data.localRepositoryPath}"`);
+    });
+
+    beforeEach(function() {
+      data.GitilesIDLFile = foam.lookup('org.chromium.webidl.GitilesIDLFile');
+      data.GithubIDLFile = foam.lookup('org.chromium.webidl.GithubIDLFile');
+      data.BlinkParser = foam.lookup('org.chromium.webidl.Parser');
```

Comment:
We can probably do:

```js
data.Parser = foam.lookup(`org.chromium.webidl.${data.parser}`);
```
Then use `data.Parser` later.

We will have to update the Blink config to set `config.parser = 'Parser'`, and add a comment that Blink uses default parser.